cray_product_catalog/catalog_update.py

```diff
@@ -172,10 +184,16 @@ def update_config_map(data: dict, name, namespace):
             LOGGER.exception("Error calling read_namespaced_config_map")
 
             # ConfigMap doesn't exist yet
-            if err.status == ERR_NOT_FOUND:
+            if err.status != ERR_NOT_FOUND:
+                raise   # unrecoverable
+            elif name == CONFIG_MAP:
```

Comment:
This should be an `if`, because the previous clause raises an exception.
```suggestion
            if name == CONFIG_MAP:
```